FilmFestTests/MovieRepoTest.swift

```diff
@@ -41,12 +41,20 @@ class MovieRepoTest: XCTestCase {
     
     func testAdd_RepositoryCount_IsOne()
     {
-        //sut.add(movie: scifiMovie)
+        sut.add(movie: scifiMovie)
         XCTAssertEqual(sut.count, 1)
     }
     
     //TODO : Write a test for the get functionality
     //      to show it returns the expected movie
+    func testGet_Movie(){
+        sut.add(movie: actionMovie)
+        sut.get(identifier: 0)
+        XCTAssertEqual(sut.count, 1)
+    
```

Comment:
This test only shows that the sut has one movie. Better would be to test that the movie returned by the get method has the same title as actionMovie